### PR TITLE
Navbar footer routing

### PR DESCRIPTION
**Pull request by ark2002**
- Added Navbar component
- Added Footer Component
- Added Routing

Preview: 

https://deploy-preview-1--get-it-noted.netlify.app/

recording

### CODE REVIEW

**Review by sahiba0915 — APPROVED** [review]
codes looks clean, good component names used, structured everything properly.

File: src/frontend/components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.footer__socials {
+    color: var(--white-color);
+    padding: 2rem;
+    font-size: 4rem;
+}
+
+hr {
+    margin: 2rem;
+    border: 0;
+    height: 4px;
```
Comment (chanchal-panpaliya) [inline src/frontend/components/Footer/Footer.css:23]: For height use vh unit

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks good . need to change - units .

File: src/frontend/components/Footer/Footer.css (hunk 1 above)
Comment (ark2002) [reply src/frontend/components/Footer/Footer.css:23]: its just an horionzal line so it doesnt need a much of height

**Pull request merged**